### ISSUE TITLE
Public hypothes.is annotations

### ISSUE THREAD

**Issue by davidar** [ipfs/archives#34]
> Public annotations on Hypothes.is are licensed under the CC0 licence (aka Public Domain). 

- https://hypothes.is/stream
- http://hypothesis.github.io/developers.hypothes.is/API.html

CC: @jbenet @bigbluehat

**Comment by davidar** [comment]
Mirrored [here](https://ipfs.io/ipfs/QmTu1VTSVHyKMope8RpYsgpaRB44Z1oT2MAjBes73JXfDt). We should be able to store this in a nicer format once IPLD happens, but for now it's just the output of

```sh
for i in `seq 0 20 68600`; do
  curl "https://hypothes.is/api/search?limit=20&offset=$i" > $i.json
done
```

**Comment by BigBlueHat** [comment]
@davidar great start! Would it be possible to turn those into individual files in IPFS named by their "id" value?

Also, I'm working on a conversion tool for Hypothesis (and Annotator) JSON to Web Annotation Data Model: http://github.com/BigBlueHat/to-web-annotation

Excited to see this starting!

**Comment by davidar** [comment]
> Would it be possible to turn those into individual files in IPFS named by their "id" value?

Yes, although ipfs currently has problems handling directories containing lots of small files (cc @whyrusleeping)

**Comment by mklarmann** [comment]
This is an awesome idea! 

I know this message doesn't bring much value. So feel free to delete it. Just wanted to bring my emotional support to this! : )
